components/camel-infinispan/src/main/java/org/apache/camel/component/infinispan/InfinispanConfiguration.java

```diff
@@ -55,6 +56,11 @@
     private boolean clusteredListener;
     @UriParam
     private InfinispanQueryBuilder queryBuilder;
+    @UriParam(label = "advanced")
```

Comment:
Use javaType = "String" so the tooling knows it can use a plain string to set multiple flags in the same string separated by comma. And write that in the javadoc as documentation.